dist/DynamicDataTable.js

```diff
@@ -105,10 +109,26 @@ class DynamicDataTable extends Component {
       }
     }
 
+    const regExpsToExclude = fieldsToExclude.filter(field => field.constructor && field.constructor === regExpsToExclude);
+
     for (let i = 0; i < fields.length; i++) {
-      const field = fields[i]; // Field exclusion
+      const field = fields[i];
+      let excluded = false; // Field exclusion
```

Comment:
```suggestion
      let shouldExclude = false; // Field exclusion
```

I think `shouldExclude` makes more sense than `excluded` as the exclusion has yet to occur.